tests/br_tikv_outage/run.sh

```diff
@@ -71,6 +71,8 @@ github.com/pingcap/br/pkg/conn/hint-get-backup-client=1*return(\"$hint_get_backu
     backup_pid=$!
     single_point_fault $failure
     wait $backup_pid
+    # let tikv start up completely if backup is finished before tikv restarts
+    ensure_tikv
```

Comment:
shall we move this line into the case block to make it clearer which failures depend on this

Reply:
I add the line into kv_outage function where tikv restart. It seems be better.